testing.rst

```diff
@@ -630,6 +630,10 @@ You can pass any
 :class:`Symfony\\Bundle\\FrameworkBundle\\Test\\TestBrowserToken` object and
 stores in the session of the test client.
 
+.. note::
+
+``loginUser()`` method is not expected to work in all of the cases. As it relies on session-based authentication, it won't work if the routes you are trying to test are protected by a stateless firewall or if you are using a custom authenticator.
```

Comment:
Please indent by 4 spaces, thanks

Reply:
Starting from ``loginUser()``? Or the `.. note::` part as well?

Reply:
Done.